tests/run_tests/driver_mag_test.py

```diff
@@ -0,0 +1,246 @@
+#  Copyright (c) 2021 Mira Geoscience Ltd.
+#
+#  This file is part of geoapps.
+#
+#  geoapps is distributed under the terms and conditions of the MIT License
+#  (see LICENSE file at the root of this source code package).
+
+from os import path
+
+import numpy as np
+from discretize.utils import active_from_xyz, mesh_builder_xyz, refine_tree_xyz
+from geoh5py.objects import Points, Surface
+from geoh5py.workspace import Workspace
+from scipy.spatial import Delaunay
+from SimPEG import utils
+
+from geoapps.utils import get_inversion_output, treemesh_2_octree
+
+
+def setup_workspace(work_dir, phys_prop):
+    project = path.join(work_dir, "mag_test.geoh5")
+    workspace = Workspace(project)
+
+    # Topography
+    xx, yy = np.meshgrid(np.linspace(-200.0, 200.0, 50), np.linspace(-200.0, 200.0, 50))
+    b = 100
+    A = 50
+    zz = A * np.exp(-0.5 * ((xx / b) ** 2.0 + (yy / b) ** 2.0))
+    topo = np.c_[utils.mkvc(xx), utils.mkvc(yy), utils.mkvc(zz)]
+    triang = Delaunay(topo[:, :2])
+
+    Surface.create(workspace, vertices=topo, cells=triang.simplices, name="topography")
+
+    # Observation points
+    xr = np.linspace(-100.0, 100.0, 20)
```

Comment:
I think we talked about a really extreme coarsening at one point.  Now that you've played around with the test, do you think we could get away with a single observation point and really coarse mesh?